dipy/io/dpy.py

```diff
@@ -85,12 +87,24 @@ def __init__(self, fname, mode='r', compression=0):
             self.track_no = len(self.offsets) - 1
             self.offs_pos = 0
 
-    def version(self):
+    def version(self) -> str:
 
         return self.f.attrs['version']
 
-    def write_track(self, track):
+    def write_track(self, track: np.array) -> None:
         """ write on track each time
+
+        Args
```

Comment:
Can you follow numpy documentation standard : https://numpydoc.readthedocs.io/en/latest/format.html

Reply:
Okay